src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 
 import Post from './Post/Post';
+import { updateNewPostTextActionCreator, addPostActionCreator } from '../../../redux/state';
```

Comment:
Consider moving this import up, outside of this "local to the file" group.

Reply:
/fxd